helm/flink-kubernetes-operator/values.yaml

```diff
@@ -183,3 +183,7 @@ operatorHealth:
   startupProbe:
     failureThreshold: 30
     periodSeconds: 10
+
+# Set exec command for the postStart hook of the main container
+postStart:
+  execCommand: []
```

Comment:
I think it would be more intuitive to provide the entire yaml content of the `postStart` section instead of `execCommand` which would require extra documentation and would limit the usage.

Reply:
Sounds good. I'll make an update